scripts/lib/install

```diff
@@ -205,6 +243,63 @@ case "$POSTGRESQL_VERSION" in
         ;;
 esac
 
+if [ -z "$PUSH_NOTIFICATIONS" ] && [ -z "$NO_PUSH_NOTIFICATIONS" ]; then
+    # Unless specified, we default to --no-push-notifications
+    NO_PUSH_NOTIFICATIONS=1
+fi
+
+# We set these to Python-style True/False string values, for easy
+# insertion into the settings.py file.
+if [ -n "$PUSH_NOTIFICATIONS" ]; then
+    SERVICE_MOBILE_PUSH="True"
+
+    # --push-notifications also enables SUBMIT_USAGE_STATISTICS, unless the user
+    # explicitly opted out by passing --no-submit-usage-statistics.
+    if [ -n "$NO_SUBMIT_USAGE_STATISTICS" ]; then
+        SERVICE_SUBMIT_USAGE_STATISTICS="False"
+    else
+        SERVICE_SUBMIT_USAGE_STATISTICS="True"
+    fi
+else
+    SERVICE_MOBILE_PUSH="False"
+    # SUBMIT_USAGE_STATISTICS without PUSH_NOTIFICATIONS wouldn't really
+    # make much sense.
+    SERVICE_SUBMIT_USAGE_STATISTICS="False"
+fi
+
+# If user asked for push notifications, prompt for ToS acceptance.
+if [ -n "$PUSH_NOTIFICATIONS" ]; then
+    echo
+    echo "You chose to register your server for the Mobile Push Notifications Service."
+    echo "Doing so will share basic metadata with the service's maintainers, including:"
```

Comment:
We know if this will share more than basic metadata -- I think we should change what we print to add a line about sharing usage statistics, if `SERVICE_SUBMIT_USAGE_STATISTICS` is `True`

Reply:
added an if here
```
    echo "    <https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html#uploading-basic-metadata>"
    if [ -z "$NO_SUBMIT_USAGE_STATISTICS" ]; then
        echo "* The server's usage statistics; see:"
        echo "    <https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html#uploading-usage-statistics>"
    fi
```